cfme/utils/events.py

```diff
@@ -185,19 +92,26 @@ def __repr__(self):
                                                              val=self.value, cmp=self.cmp_func)
 
 
-# fixme: would it be better to create event prototype and just clone it ?
 class Event(object):
+    """ Event represents either event received by REST API or an expected event.
+
+    :var DEFAULT_ATTRS: List of default attributes to process.
     """
-    represents either db event received by CFME and stored in event_streams or an expected event
-    """
+    DEFAULT_ATTRS = [
```

Comment:
What's the reason for static list of attributes ? Can you get them dynamically ?

Reply:
@izapolsk I'll revise once again because I had some reason for this.